app/components/certificate/description.tsx

```diff
@@ -16,10 +27,21 @@ export default function DescriptionSection({
   validToFormatted,
   description,
   isRenewable,
+  link,
 }: DescriptionSectionProps) {
   return (
-    <Flex flexDirection="column" gap="3" marginTop={{ base: 1, md: 5 }} fontSize="md">
-      <Text maxW={600}>{description}</Text>
+    <Flex flexDirection="column" gap="3" fontSize="md">
+      <Text maxW={600}>
+        {description}
+        {link && (
+          <>
+            &nbsp;To learn more, refer to this&nbsp;
+            <Link as={RemixLink} to={link}>
```

Comment:
It's not a big deal, but I think it's good to align with other page sentence. `To learn more, refer to our information page`